xbmc/linux/LinuxTimezone.h

```diff
@@ -59,6 +60,7 @@ class CLinuxTimezone : public ISettingCallback, public ISettingsHandler
    std::map<std::string, std::string> m_countriesByTimezoneName;
 };
 
-extern CLinuxTimezone g_timezone;
+XBMC_GLOBAL_REF(CLinuxTimezone,g_timezone);
+#define g_timezone XBMC_GLOBAL_USE(CLinuxTimezone)
```

Comment:
Much more important here: Do you have a better idea to get rid of this global nogoes completely? They will kick us in the balls later on again and a proper design would help use a lot here.